src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        String[] colors = {"white", "black", "orange", "yellow", "grey", "blue"};
```

Comment:
The array of colors could be extracted to a constant field, as it represents a fixed set of possible values. This would comply with the best practice of avoiding magic numbers or magic values.